### ISSUE TITLE
aligncheck doesn't support vendoring

### ISSUE THREAD

**Issue by stevenh** [opennota/check#33]
If you try to use aligncheck on code which has vendored dependencies it reports bogus errors

For example:
```
main.go:6:2:warning: could not import github.com/pkg/errors (cannot find package "github.com/pkg/errors" in any of: (aligncheck)
main.go:11:14:warning: undeclared name: errors (aligncheck)
```
Where main.go clearly imports errors:
```go
package main

import (
        "fmt"

        "github.com/pkg/errors"
)

func main() {
        err := fmt.Errorf("some error")
        fmt.Println(errors.Wrap(err, "my wrapped error"))
}
```

To ensure you get this error you need github.com/pkg/errors vendored and it must NOT be in your GOPATH.

Also reported to gometalinter:  https://github.com/alecthomas/gometalinter/issues/190

**Comment by stevenh** [comment]
Fixed now
